packages/ffe-account-selector-react/src/subcomponents/input-field/InputField.js

```diff
@@ -89,17 +97,24 @@ class Input extends Component {
                 {showReset && (
                     <button
                         className="ffe-base-selector__reset-button"
-                        onMouseDown={e => {
-                            e.preventDefault();
-                            onReset();
-                        }}
+                        onMouseDown={this.onReset}
                         tabIndex={-1}
                         type="button"
-                        aria-label="Reset"
                     >
-                        <KryssIkon className="ffe-base-selector__reset-button-icon" />
+                        <KryssIkon className="ffe-base-selector__reset-button-icon"/>
                     </button>
                 )}
+                <button
```

Comment:
Jeg tenker denne knappen ikke brude vises om `showReset === true`, ellers blir det to synlige knapper.

Reply:
`showReset === true` når det finnes karakterer i inputfeltet. Det burde også være mulig å ekspandere og kollapse når det finnes karakterer i input feltet.